package/doc/sphinx/source/index.rst

```diff
@@ -73,69 +73,6 @@ use the MDAnalysis library which may be of interest.
 
 .. _installation-instructions:
 
-Installing MDAnalysis
```

Comment:
There's still the anchor `.. _installation-instructions:` present. Move it above line 61 so that any links will still lead to a place from whence users can find installation instructions.